actors/miner/tests/batch_method_network_fees_test.rs

```diff
@@ -175,7 +175,8 @@ fn insufficient_funds_for_batch_precommit_in_combination_of_fee_debt_and_network
 #[test]
 fn enough_funds_for_fee_debt_and_network_fee_but_not_for_pcd() {
     let actor = ActorHarness::new(*PERIOD_OFFSET);
-    let rt = actor.new_runtime();
+    let mut rt = actor.new_runtime();
+    rt.policy.new_miner_deposit = TokenAmount::default();
```

Comment:
Why setting this unrealistic amount? Tests need to be updated to use a real deposit.

Also `default` is a rather opaque way of saying `zero`. Please use `zero()` everywhere so it's more obvious that this is _not a default value_, but a fake one.